docs/trace_collection.md

```diff
@@ -0,0 +1,178 @@
+    ```
+
+3. Datadog tracer implements the [Open Tracing standard][8]. Configure and register the `DDTracer` globally as Open Tracing `Global.sharedTracer`. You only need to do it once, usually in your `AppDelegate` code:
+
+    ```swift
+    import Datadog
+    import OpenTracing
+
+    Global.sharedTracer = DDTracer.initialize(
+        configuration: DDTracer.Configuration(
+            sendNetworkInfo: true
+        )
+    )
+    ```
+
+4. Instrument your code using the following methods:
+
+    ```swift
+    import OpenTracing
+
+    let span = Global.sharedTracer.startSpan(operationName: "<span_name>")
+    // do something you want to measure ...
+    // ... then, when the operation is finished:
+    span.finish()
+    ```
+
+5. (Optional) - Set child-parent relationship between your spans:
+
+    ```swift
+    let responseDecodingSpan = Global.sharedTracer.startSpan(
+        operationName: "response decoding",
+        childOf: networkRequestSpan.context // make it a child of `networkRequestSpan`
+    )
+    // ... decode HTTP response data ...
+    responseDecodingSpan.finish()
+    ```
+
+6. (Optional) - Provide additional tags alongside your span:
+
+    ```swift
+    span.setTag(key: "http.url", value: url)
+    ```
+
+7. (Optional) Attach an error to a span - you can do so by logging the error information using the [standard Open Tracing log fields][9]:
+
+    ```swift
+    span.log(
+        fields: [
+            "event": "error",
+            "error.kind": "I/O Exception",
+            "message": "File not found",
+            "stack": "FileReader.swift:42",
+        ]
+    )
+    ```
+
+8. (Optional) To distribute traces between your environments, for example frontend - backend, inject tracer context in the client request:
+
+    ```swift
+    import OpenTracing
+    import Datadog
+
+    var request: URLRequest = ... // the request to your API
+
+    let span = Global.sharedTracer.startSpan(operationName: "network request")
+
+    let headersWritter = DDHTTPHeadersWriter()
+    Global.sharedTracer.inject(spanContext: span.context, writer: headersWritter)
+
+    for (headerField, value) in headersWritter.tracePropagationHTTPHeaders {
+        request.addValue(value, forHTTPHeaderField: headerField)
+    }
+    ```
+    This will set additional tracing headers on your request, so that your backend can extract it and continue distributed tracing. If your backend is also instrumented with [Datadog APM & Distributed Tracing][10] you will see the entire front-to-back trace in Datadog dashboard.
+
+
+## Batch collection
+
+All the spans are first stored on the local device in batches. Each batch follows the intake specification. They are sent as soon as network is available, and the battery is high enough to ensure the Datadog SDK does not impact the end user's experience. If the network is not available while your application is in the foreground, or if an upload of data fails, the batch is kept until it can be sent successfully.
```

Comment:
I see 👌. How about this:
```suggestion
All the spans are first stored on the local device in batches. Each batch follows the intake specification. They are sent periodically if network is available, and the battery is high enough to ensure the Datadog SDK does not impact the end user's experience. If the network is not available while your application is in the foreground, or if an upload of data fails, the batch is kept until it can be sent successfully.
```

Reply:
👍